corehq/apps/cloudcare/templates/cloudcare/formplayer_home.html

```diff
@@ -12,7 +12,7 @@
 {% block head %}
   {{ block.super }}
   <script src="{% static 'datatables/media/js/jquery.dataTables.min.js' %}"></script>
-  <script src="{% static 'datatables/media/js/dataTables.bootstrap.min.js' %}"></script>
+  <script src="{% static 'datatables-bootstrap3/BS3/assets/js/datatables.js' %}"></script>
```

Comment:
Do you know what datatables is used for in web apps? Just curious.

Reply:
no idea, but it was 404ing so that's all I fixed